pilot/pkg/features/pilot.go

```diff
@@ -288,6 +288,13 @@ var (
 		"If this is set to true, support for Kubernetes service-apis (github.com/kubernetes-sigs/service-apis) will "+
 			" be enabled. This feature is currently experimental, and is off by default.").Get()
 
+	EnableVirtualServiceDelegate = env.RegisterBoolVar(
+		"PILOT_ENABLE_VIRTUAL_SERVICE_DELEGATE",
```

Comment:
Will this be exposed via MeshConfig?

Reply:
It should be default eventually, its just set as a feature flag to ensure we don't break anything while its still being stabilized. I don't think we need to commit it to a long term API?

Reply:
I get that but what's interface for user's to turn it on 1.6? Can you transparently pass ENV variables via istioctl or operator APIs?